### PR TITLE
clear cache after generating resource

### PR DESCRIPTION
**Pull request by hairmare**
This might help with cache problems introduced by libgraviton/graviton#45.

We still need to find out of those really are an issue and if this is best put here. I'm also not sure if I should make this an option or not.

Feedback welcome ;)

### CODE REVIEW

File: src/Graviton/GeneratorBundle/Command/GenerateResourceCommand.php [hunk 1]
```diff
@@ -64,6 +70,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
 
+        $commandInput = new ArrayInput(array(
+            'command' => 'cache:clear',
+            '--no-warmup' => true
+        ));
+        $command = $this->getApplication()->find('cache:clear');
+        if ($command->run($commandInput, $output) == 0) {
+            $output->isVerbose() && $output->writeln(
+                'cache cleared'
+            );
```
Comment (lapistano) [inline src/Graviton/GeneratorBundle/Command/GenerateResourceCommand.php]: I find the formatting quite hard to read.

```php
if ($command->run($commandInput, $output) == 0) {
    $output->isVerbose() && $output->writeln('cache cleared');
}
```
would be easier for me to read.

Comment (hairmare) [inline src/Graviton/GeneratorBundle/Command/GenerateResourceCommand.php]: changed

**Comment by hairmare** [comment]
@narcoticfresh do we need this? I would add some tests and remove WIP if its needed but the need seems to have gone away...

**Comment by hairmare** [comment]
Not needed...

**Pull request closed without merging**